project.clj

```diff
@@ -1,25 +1,21 @@
-(def jruby-version "9.4.2.0")
+(def jruby-version "9.4.3.0")
 
-(defproject puppetlabs/jruby-deps "9.4.2.0-2-SNAPSHOT"
+(defproject puppetlabs/jruby-deps "9.4.3.0-1-SNAPSHOT"
   :description "JRuby dependencies"
   :url "https://github.com/puppetlabs/jruby-deps"
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0"}
 
+  :parent-project {:coords [puppetlabs/clj-parent "7.1.3"]
```

Comment:
We just released 7.2.0, did you want to pull that in here?

Reply:
yeah

Reply:
updated